tests/CMakeLists.txt

```diff
@@ -233,38 +247,18 @@ if(AVIF_BUILD_APPS)
         endif()
     endif()
     target_link_libraries(are_images_equal aviftest_helpers avif_enable_warnings)
-    add_test(NAME test_cmd COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd.sh ${CMAKE_BINARY_DIR}
-                                   ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_animation COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_animation.sh ${CMAKE_BINARY_DIR}
-                                             ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_grid COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_grid.sh ${CMAKE_BINARY_DIR}
-                                        ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_icc_profile COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_icc_profile.sh ${CMAKE_BINARY_DIR}
-                                               ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_lossless COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_lossless.sh ${CMAKE_BINARY_DIR}
-                                            ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_metadata COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_metadata.sh ${CMAKE_BINARY_DIR}
-                                            ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_progressive COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_progressive.sh ${CMAKE_BINARY_DIR}
-                                               ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_stdin COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_stdin.sh ${CMAKE_BINARY_DIR}
-                                         ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
-    add_test(NAME test_cmd_targetsize COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_targetsize.sh ${CMAKE_BINARY_DIR}
-                                              ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
+    add_cmd_test(test_cmd ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_animation ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_grid ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_icc_profile ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_lossless ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_metadata ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_progressive ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_stdin ${CMAKE_CURRENT_SOURCE_DIR}/data)
+    add_cmd_test(test_cmd_targetsize ${CMAKE_CURRENT_SOURCE_DIR}/data)
 
     if(AVIF_ENABLE_JPEG_GAIN_MAP_CONVERSION)
-        add_test(NAME test_cmd_avifgainmaputil COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd_avifgainmaputil.sh
-                                                       ${CMAKE_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/data
-        )
+        add_cmd_test(test_cmd_avifgainmaputil ${CMAKE_CURRENT_SOURCE_DIR}/data)
```

Comment:
IMPORTANT: The `AVIFGAINMAPUTIL` variable defined in test_cmd_avifgainmaputil.sh needs to be updated:

```
source $(dirname "$0")/cmd_test_common.sh || exit

AVIFGAINMAPUTIL="${BINARY_DIR}/avifgainmaputil"
```

We can either update `test_cmd_avifgainmaputil.sh` or move the definition of `AVIFGAINMAPUTIL` to cmd_test_common.sh.

Please test manually on Linux with the "Ninja Multi-Config" generator. (No need to add "Ninja Multi-Config" to a CI workflow in this pull request.)

Reply:
Done